_posts/2017-02-27-rgsoc.md

```diff
@@ -0,0 +1,53 @@
+---
+layout:     post
+title:      Rails Girls Summer of Code + Servo
+date:       2017-02-27 01:30:00
+summary:    Building a more diverse open source ecosystem.
+categories:
+---
+Servo is taking part in the [Rails Girls Summer of Code](http://railsgirlssummerofcode.org/) program as a
+mentoring project once again. We're really excited about the program after our wonderful experience last year! 
+
+## What is Rails Girls Summer of Code?
+
+The Rails Girls Summer of Code program (RGSoC) takes place each year between July and September. Developers form pairs and
```

Comment:
This might be nitpicking, but I'm not sure about the "_Developers_ form pairs.." part.

I think I'd rather say something along the lines of
> Applicants form pairs ("teams") and [...]